app/src/main/java/com/android/solvit/provider/ui/profile/ProviderProfile.kt

```diff
@@ -71,508 +69,258 @@ import com.android.solvit.shared.ui.navigation.Screen
+                                    tint = colorScheme.primary,
+                                    modifier = Modifier.size(32.dp))
+                                Text(
+                                    text = "${provider.rating}",
+                                    color = colorScheme.onBackground,
+                                )
+                              }
+                          Row(
+                              horizontalArrangement = Arrangement.spacedBy(8.dp),
+                              modifier = Modifier.testTag("PopularityItem")) {
+                                Text(
+                                    text = "Popular",
+                                    color = colorScheme.primary,
+                                    fontWeight = FontWeight.Bold,
+                                    fontSize = 18.sp)
+                                Text(text = "${provider.popular}", color = colorScheme.onBackground)
+                              }
+                          Row(
+                              horizontalArrangement = Arrangement.spacedBy(8.dp),
+                              modifier = Modifier.testTag("EarningsItem")) {
+                                Icon(
+                                    painter = painterResource(R.drawable.money_icon),
+                                    contentDescription = "Earnings",
+                                    tint = colorScheme.primary,
+                                    modifier = Modifier.size(32.dp))
+                                Text(text = "$earnings CHF", color = colorScheme.onBackground)
+                              }
+                        }
+                    Column(
+                        modifier = Modifier.fillMaxWidth().testTag("TasksColumn"),
+                        horizontalAlignment = Alignment.CenterHorizontally) {
+                          Text(
+                              text = "Jobs",
+                              color = colorScheme.primary,
+                              fontWeight = FontWeight.Bold,
+                              fontSize = 18.sp)
+                          Column {
+                            for (status in ServiceRequestStatus.entries) {
+                              val size =
+                                  when (status) {
+                                    ServiceRequestStatus.PENDING -> filteredPendingTasks.size
+                                    ServiceRequestStatus.ACCEPTED -> filteredAcceptedTasks.size
+                                    ServiceRequestStatus.SCHEDULED -> filteredScheduledTasks.size
+                                    ServiceRequestStatus.COMPLETED -> filteredCompletedTasks.size
+                                    ServiceRequestStatus.CANCELED -> filteredCanceledTasks.size
+                                    ServiceRequestStatus.ARCHIVED -> filteredArchivedTasks.size
+                                  }
```

Comment:
It seems like the number of requests for each status is not updated dynamically. When managing requests in the request dashboard of the provider, the modifications are not reflected in the profile display.

Reply:
This issue is caused by the fact that we use different instances of the `ServiceRequestViewModel` in the job dashboard and in the profile screen so the changes are not instantly synced  (this is one of the reason we should use the same instance of the ViewModels). However I fix this in my other PR.

Reply:
👍 it's fixed !